README.md

```diff
@@ -233,7 +233,7 @@ c.GlobusOAuthenticator.identity_provider = 'globusid.org'
 ```
 
 If you want to use a _Linked Identity_ such as `malcolm@universityofindependence.edu`,
-go to your [App Developer page](http://www.developers.globus.org) and set
+go to your [App Developer page](http://developers.globus.org) and set
 *Required Identity Provider* for your app to _<Your University>_, and set the
```

Comment:
On the very next line there is another minor problem, the \<\> cause `Your University` to not display correctly and should be removed.